features/page_objects/back_office/new/user_accept_invite_page.rb

```diff
@@ -0,0 +1,10 @@
+require_relative "sections/govuk_banner.rb"
+
+class UserAcceptInvitePage < SitePrism::Page
+  section(:govuk_banner, GovukBanner, GovukBanner::SELECTOR)
+
+  element :password_field, "#user_password"
```

Comment:
I've always used parentheses in the past for elements, e.g. `element(:password_field, "#user_password")`. I don't have an opinion on whether it's better to have them or not, but is it possible to include them for consistency?

Reply:
sure :)